flink-contrib/flink-storm-compatibility/flink-storm-compatibility-examples/src/main/java/org/apache/flink/stormcompatibility/wordcount/StormWordCountLocal.java

```diff
@@ -67,8 +67,10 @@ public static void main(final String[] args) throws Exception {
 
 		Utils.sleep(10 * 1000);
 
-		// TODO kill does no do anything so far
 		cluster.killTopology(topologyId);
+		// killing sends STOP signal, takes some time to clean up
+		Utils.sleep(1000);
```

Comment:
Fair enough. But why shouldn't we allow Flink to properly wait for a job termination? After you've sent the stop signal you could send the JM another message which requests a notification once the job has terminated?